docs/interface_specification.md

```diff
@@ -62,71 +81,78 @@ Illegal Instruction
     - Support for multiple fetch entries in a single packet. Only useful for multiple-issue BE implementations.
 
 ### BE->FE (fe\_cmd)
-- Description: FE Command Queue (fe\_cmd) sends the following commands from the Back End to the Front End, using a single command queue containing structures of type bp\_fe\_cmd\_s. From the perspective of architectural and microarchitectural state, enqueuing a command onto the command queue is beyond the point of no return, although the side-effects are not guaranteed until after the fence line goes high. With the exception of the attaboy commands, the BE will not enqueue additional commands until all prior commands have been processed. The FE must dequeue attaboys immediately upon reception and should process other commands as quickly as possible. However, due to the fence line it is not an invariant that the FE processes other instructions within a single cycle. The Command Queue shall transmit a union data structure, bp\_fe\_cmd\_s which contains opcodes and operands.
```

Comment:
I un-indented this block to move the "description", "interface", and sibling top-level bullets to headings. Reviewing will be easier if you check "hide whitespace changes" from the gear icon menu in the PR header.